offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/product/MaintainProductsController.groovy

```diff
@@ -82,21 +82,21 @@ class MaintainProductsController {
         static Product createProduct(ProductCategory category, String description, String name, double unitPrice, ProductUnit unit){
             Product product
             switch (category) {
-                case "Data Storage":
+                case "DATA_STORAGE":
                     //todo do we want to set the id manually to null or update the DTO constructor?
```

Comment:
You have chosen da way
```suggestion
```

Reply:
I would keep it until the issue is removed :)

Reply:
alrighty